src/properties/int_property.rs

```diff
@@ -177,11 +177,13 @@ impl ByteProperty {
     pub(crate) fn read<R: Read + Seek>(
         cursor: &mut R,
         include_header: bool,
-        mut suggested_length: Option<u64>,
+        mut suggested_length: Option<u32>,
     ) -> Result<Self, Error> {
         let mut name = None;
         if include_header {
-            let length = cursor.read_u64::<LittleEndian>()?;
+            let length = cursor.read_u32::<LittleEndian>()?;
+            let unknown = cursor.read_u32::<LittleEndian>()?;
```

Comment:
This is probably `array_index`

See: [`Runtime/CoreUObject/Public/UObject/PropertyTag.h`](https://github.com/EpicGames/UnrealEngine/blob/3abfe77d0b24a6d8bacebd27766912e5a5fa6f02/Engine/Source/Runtime/CoreUObject/Public/UObject/PropertyTag.h#L29)